iqama_fetcher.py

```diff
@@ -0,0 +1,60 @@
+import re
+import requests
+import json
+
+_SCRAPER_URL = 'https://islam-buddy-staging.herokuapp.com/iqama'
+
+def GetIqamaTime(desired_prayer,masjid):
+  """Gets the iqama time from the heruko scraper service by peforming a GET.
+
+  Args:
+    masjid: a string representing the masjid of intereset
+
+  Returns: a dict containing the prayers and iqama times
+  """
+
+  # set up the parameters in the format expected by the scraper service
+  get_params = {
+    'masjid': masjid,
+  }
+  print 'scraper get_params = ', get_params
+  request = requests.post(_SCRAPER_URL, params=get_params, timeout=15)
+  
+  # checks if request isn't bad
+  if(request.status_code == requests.codes.ok):
+    response = json.loads(request.text)
+    print 'response from scraper service = ', response
+
+    # checks if the size of the prayer and iqama time lists are equivalent
+    if len(response[0]['Prayer']) != len(response[0]['IqamaTime']):
+      print 'Prayer and IqamaTime lists don\'t match'
+      return None
+
+    for idx, prayer in enumerate(response[0]['Prayer']):
```

Comment:
bro google maps api returns lists for daaaaaaaaays .. dunno what is your standard for bad API design .. but will look into it for youuuuu